### ISSUE TITLE
Drag & drop block on Richtext break it

### ISSUE THREAD

**Issue by MKlblangenois** [storyblok/storyblok#552]
**The issue occurs while working with:**
- [x] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other:*

**Current behavior:**
When we've the possibility to add block on Richtext, the block breaks when we drag and drop it.

**Expected behavior:**
The whole block should be able to be moved within the RichText.

**Steps to reproduce:**

![current behavior](https://i.ibb.co/RC33xm2/ezgif-2-d82303507ed9.gif)

**Comment by bstrdChronicles** [comment]
Yeah, sadly a ticket #357 i created in july that described that and some other misbehaviours of the richt text editor wasnt handled yet :/

would be so cool if it was enhanced soon.

**Comment by alvarosabu** [comment]
Hi @MKlblangenois, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
